SonarSource/sonarcloud-github-c-cpp
# SCCPPGHA-8  Make use of the Linux ARM64 CLI Scanner

List of checkpoint to verify/do before merging a PR.

### Dev checklist

* [ ] Write/update tests
* [ ] Functional validation
* [ ] Public documentation update
* [ ] Private documentation update
* [ ] Clean commits (should start with a ticket number, clear message, no fixup, wip)
* [ ] Maximize code coverage as much as possible

---

### Reviewer checklist

* [ ] Functional validation
* [ ] Check commits are clean

---

### Steps to validate this PR

Fill-in how you performed the validation, and how the reviewer can replicate it:
* [ ] Run on fork of sonar-cfamily-examples
linux [here](https://github.com/enzo-pellegrini/linux-autotools-gh-actions-sc/actions/runs/9710759349) for Linux X64 and ARM64.
 windows [here](https://github.com/enzo-pellegrini/windows-msbuild-gh-actions-sc/actions/runs/9711169911)
 macos [here](https://github.com/enzo-pellegrini/macos-xcode-gh-actions-sc/actions/runs/9711317855/job/26803816129). Notice how an ARM64 jre is provisioned, where in the previous run an x64 was provisioned. This means that performance is potentially improved, both for us and for other analyzers that could get run
